styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -302,136 +390,3 @@
     color: $brand-primary;
   }
 }
-
-
-//solutions
```

Comment:
This solutions section is what's removed. I'm not sure why it looks like i deleted quadrant and then added it but this solutions bit is all im kicking out.